.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Interesting. What is this file?

Reply:
@yasskin If you needed to add this setting, you may be able to add it to your IDE's settings instead. I think adding settings here will enable them for everyone running the project in VSCode.